### PR TITLE
add folder precourse

### PR DESCRIPTION
**Pull request by ElenaErodrial**

### CODE REVIEW

File: staff/elena-rodriguez/precourse/airlinesPro/index.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        console.log('Origen: ' + flights[op].from + '; Destino: ' + flights[op].to + '; Precio: ' + flights[op].cost + '€.')
+    }
+}
+}
+
+function lastFlights(flights) {
+console.log('Últimos vuelos del día: ')
+for (let op = 6; op < flights.length; op++) {
+    console.log('Vuelo con origen: ' + flights[op].from + ' y con destino: ' + flights[op].to)}
+}
```
Comment (ghost) [inline staff/elena-rodriguez/precourse/airlinesPro/index.js]: This function could only work correctly if you have 10 flights in the array, maybe you could use the **slice** method
example:
`var lastFlights = flights.slice(-5)`

File: staff/elena-rodriguez/precourse/airlinesPro/index.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+lastFlights(flights);
+}
+
+airlinesInfo (flights)
+
+
+function precioMenor(flights) {
+
+    var question3 = prompt('Introduce el importe')
+    var realImporte = Number(question3)
```
Comment (ghost) [inline staff/elena-rodriguez/precourse/airlinesPro/index.js]: You forgot to put a semicolon at the end of the sentence.

File: staff/elena-rodriguez/precourse/airlinesPro/index.js (hunk 1 above)
Comment (ElenaErodrial) [reply staff/elena-rodriguez/precourse/airlinesPro/index.js]: Perfect, fixed!!

File: staff/elena-rodriguez/precourse/airlinesPro/index.js (hunk 2 above)
Comment (ElenaErodrial) [reply staff/elena-rodriguez/precourse/airlinesPro/index.js]: Yes!!! Thanks!!! Fixed

**Pull request merged**